google-cloud-logging/src/main/java/com/google/cloud/logging/LogEntry.java

```diff
@@ -133,14 +131,44 @@ public Builder setResource(MonitoredResource resource) {
     /**
      * Sets the time at which the event described by the log entry occurred, in milliseconds. If
      * omitted, the Logging service will use the time at which the log entry is received.
+     *
+     * @deprecated This method is no longer recommended to setup the entry timestamp.
+     *     <p>Use {@link setTimeStamp(Instant)} instead.
```

Comment:
nit: Timestamp (small s)

Reply:
Yes, because the "timestamp" here is not a reference to the type.